common/src/View/RenderView.cpp

```diff
@@ -185,6 +185,22 @@ void RenderView::wheelEvent(QWheelEvent* event)
   update();
 }
 
+bool RenderView::event(QEvent* event)
+{
+  // Unfortunately, QWidget doesn't define a specialized handler for QNativeGestureEvent,
+  // so we must override the main event handler to handle it.
+  if (event->type() == QEvent::NativeGesture)
+  {
+    QNativeGestureEvent* gestureEvent = static_cast<QNativeGestureEvent*>(event);
```

Comment:
Please use `auto` whenever possible. I know it's controversial, but it's the style that I use now, and I clean up every file that I touch to follow this practice.